source/config/Config.cpp

```diff
@@ -1657,6 +1652,11 @@ bool PlainConfig::SampleShadow::Validate() const
         }
         else
         {
+            LOGM_ERROR(
+                Config::TAG,
+                "*** %s: Invalid file path passed for argument: %s ***",
+                DeviceClient::DC_FATAL_ERROR,
+                JSON_SAMPLE_SHADOW_INPUT_FILE);
```

Comment:
You could print the value of the `shadowInputFile` to help the user more quickly identify the problem. Same goes for `shadowOutputFile`.

Reply:
updated the log statements.